shopify/product-reviews-template/api/actions/sendEmail.js

```diff
@@ -0,0 +1,43 @@
+import { SendEmailGlobalActionContext } from "gadget-server";
+import React from "react";
+import { render } from "@react-email/render";
+import { Container, Button } from "@react-email/components";
+
+/**
+ * @param { SendEmailGlobalActionContext } context
+ */
+export async function run({
+  params,
+  logger,
+  api,
+  connections,
+  emails,
+  currentAppUrl,
+}) {
+  const { singleUseCode, email } = params;
+
+  await emails.sendMail({
+    to: email,
+    subject: "Review your purchase",
+    html: await render(
+      <Container>
+        {/* Add more text in here */}
+        <Button
+          href={`${currentAppUrl}/review/${singleUseCode}`}
```

Comment:
This should be documented in our template docs!

Reply:
Added a note in the code and going to update the docs for this